### ISSUE TITLE
html not rendered correctly with panic 2.2.3

### ISSUE THREAD

**Issue by rkrug** [rstudio/rstudio#3274]
### System details

    RStudio Edition : Desktop
    RStudio Version : any
    OS Version      : macOS High Sierra
    R Version       : 3.5.1

### Steps to reproduce the problem
Open a new R Notebook, compile as html

### Describe the problem in detail
The front matter is ignored and displayed as text in the html, and panic gives a warning:
"[WARNING] This document format requires a nonempty <title> element.   Please specify either 'title' or 'pagetitle' in the metadata."

### Describe the behavior you expected
the html iOS not rendered correctly

this happens with panic version 2.2.3, but works with panda version 2.2.1

**Comment by ronblum** [comment]
@rkrug Thanks for reporting this. Can you please provide an example so that we can try to reproduce this?

**Comment by rkrug** [comment]
@jjallaire Yup - that sounds like it. Will try with the new panic as soon as it is in `homebrew`
